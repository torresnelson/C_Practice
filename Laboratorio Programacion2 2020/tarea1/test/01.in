esVaciaPila
apilar 15 5.2
esVaciaPila
cima
desapilar
esVaciaPila
apilar 95 24.0
cima
apilar 85 2.0
cima
desapilar
cima
apilar 65 3.0
apilar 25 67.0
apilar 35 -12.0
apilar 75 2.5
apilar 45 2.1
apilar 55 -24.0
cima
desapilar
desapilar
desapilar
desapilar
desapilar
cima
esVaciaPila
Fin
