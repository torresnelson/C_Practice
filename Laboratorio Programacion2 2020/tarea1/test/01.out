1>La pila est치 vacia. 
2>Apilar. 
3>La pila no est치 vacia. 
4>(15,5.20)
5>Desapilar. 
6>La pila est치 vacia. 
7>Apilar. 
8>(95,24.00)
9>Apilar. 
10>(85,2.00)
11>Desapilar. 
12>(95,24.00)
13>Apilar. 
14>Apilar. 
15>Apilar. 
16>Apilar. 
17>Apilar. 
18>Apilar. 
19>(55,-24.00)
20>Desapilar. 
21>Desapilar. 
22>Desapilar. 
23>Desapilar. 
24>Desapilar. 
25>(65,3.00)
26>La pila no est치 vacia. 
27>Fin.
